.github/workflows/ci.yml

```diff
@@ -558,7 +558,8 @@ jobs:
           - test_name: lit-set-heartbeat-timeout
           - test_name: lit-vc-test
           - test_name: lit-identity-test
```

Comment:
:nail_care: maybe we should rename this one to `lit-substrate-ii-test` then :thinking:

Reply:
`ii` means invocation identity?

Reply:
"indirect invocation" (as opposed to "di")